cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -55,6 +55,17 @@ public int config() throws Exception {
 
     @Command(name = "clone")
     public int cloneCmd(@Parameters(index = "0") String url) {
+        File configFilePath = getConfigFilePath(); // Get path for config file 
+
+        try {
+            if (!configFilePath.exists()) {// Check if config file exist
+                defaultConfig();
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
should not print stakctraces raw to the user. and what do we expect that can throw an exception here?

Reply:
I thought the code block might throw an exception, but I've fixed that now.